[CI] Remove CI skip list for inductor integration tests

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #113668
* #113559
* __->__ #113446
* #113575
* #113574

Summary: Switch to completely rely on checking against expected result files.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

desertfire committed:
[CI] Remove CI skip list for induct integration tests

Summary: Switch to completely rely on checking against expected result files.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for induct integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

ezyang commented on benchmarks/dynamo/ci_expected_accuracy/inductor_timm_training.csv:
```diff
@@ -126,10 +134,18 @@ mobilenetv3_large_100,pass,8
+mobilevit_s,pass,8
+
+
+
```
Why are there so many newlines in the csv files

malfet replied:
This is to prevent fake merge conflict when co-dev PR and something else modify those files: having 3 empty lines ensures that patches can applied in any order even if they made to the adjacent lines

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire commented on benchmarks/dynamo/ci_expected_accuracy/aot_inductor_torchbench_inference.csv:
```diff
@@ -250,8 +250,8 @@ vgg16,pass,0
 
-yolov3,fail_to_run,0
+vision_maskrcnn,fail_to_run,0
 
 
 
-vision_maskrcnn,fail_to_run,0
+yolov3,fail_to_run,0
```
`update_expected.py` did that automatically.

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Remove CI skip list for inductor integration tests"


Summary: Switch to completely rely on checking against expected result files.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]
